### PR TITLE
Ensure Getting Started examples are compliant with Pegasus

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Also updated references to Ocean classes to exploit integration with tools' repos.

### CODE REVIEW

File: docs/examples/hybrid_solver_service.rst [hunk 1]
```diff
@@ -60,7 +62,7 @@ Formulate the Problem
 =====================
 
 For a social graph, `G`, this example simply builds a random sparse graph---using the
-`NetworkX <https://networkx.github.io/>`_ `random_geometric_graph()` function, which
+`NetworkX <https://networkx.github.io/>`_ :code:`random_geometric_graph()` function, which
```
Comment (arcondello) [inline docs/examples/hybrid_solver_service.rst:65]: why not `:func:`random_geomatric_graph`?

Comment (JoelPasvolsky) [reply docs/examples/hybrid_solver_service.rst:65]: Using `:func:`random_geomatric_graph`` results in what looks like a link but is not:
image

I can link it using `:func:`~networkx.generators.geometric.random_geometric_graph()``. I've made the change. It has the disadvantage that if NetworkX restructures, it breaks.

Comment (arcondello) [reply docs/examples/hybrid_solver_service.rst:65]: I _think_ ```:func:`~networkx.random_geometric_graph` ``` should work

Comment (JoelPasvolsky) [reply docs/examples/hybrid_solver_service.rst:65]: Unfortunately not

Comment (randomir) [reply docs/examples/hybrid_solver_service.rst:65]: I'd encourage linking with function docs even for 3rd party packages. We mention NetworkX functions below in a few places as well.

File: docs/examples/map_kerberos.rst [hunk 2]
```diff
@@ -84,10 +86,12 @@ Ocean's :doc:`dwave_networkx </docs_dnx/sdk_index>` can return a
 `minimum vertex coloring <https://en.wikipedia.org/wiki/Graph_coloring>`_ for a graph,
 which assigns a color to the vertices of a graph in a way that no adjacent vertices
 have the same color, using the minimum number of colors. Given a graph representing a
-map and a :term:`sampler`, the `min_vertex_coloring` function tries to solve the
+map and a :term:`sampler`, the :code:`min_vertex_coloring` function tries to solve the
```
Comment (randomir) [inline docs/examples/map_kerberos.rst:89]: Instead of just formatting as code, I would link with dwave-networkx docs.

**Review by randomir — APPROVED** [review]
LGTM.

**Comment by JoelPasvolsky** [comment]
I think I've updated for all GS examples to directly link

**Pull request merged**